spec/plugins/data_table/server_side.js

```diff
@@ -894,6 +894,29 @@ describe("DataTable Plugin", function() {
         }
       });
     });
-  });
+    
+    describe("_fetchCSV", function () {
+      it("should have correct filters in parameters", function () {
```

Comment:
Nit. Just the name of this test is a missing a little (at least to me). "it should append current filter parameters to the URL when requesting a CSV export."